src/org/labkey/test/components/ui/edit/EditInlineField.java

```diff
@@ -33,14 +33,23 @@ public WebDriver getDriver()
         return _driver;
     }
 
-    public void setValue(String value)
+    public void setValue(String value, boolean skipEnter)
     {
         open();
         WebElement input = elementCache().input;
         // 'setFormElement' calls 'WebElement.clear()' which can close the edit-in-place input
         getWrapper().actionClear(input);
-        input.sendKeys(value, Keys.ENTER);
-        getWrapper().shortWait().until(ExpectedConditions.stalenessOf(input));
+
+        if (!skipEnter)
```

Comment:
If `skipEnter` is true, this is effectively just clearing the input, isn't it? That seems odd. Seems like it would be better to have a `clearValue` method.